extension/package.json

```diff
@@ -5,14 +5,16 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "check-coverage": "nyc check-coverage --lines 89",
+    "test-coverage-run": "c8 npm run test-run && npm run check-coverage",
+    "test-run": "npm run unit && npm run integration && npm run e2e",
+    "check-coverage": "c8 check-coverage --lines 89",
     "test": "npm run check && npm run cover:unit && npm run cover:integration && npm run cover:e2e && npm run check-coverage",
```

Comment:
Is this command used anywhere? If not, remove it.

Reply:
I think this will be helpful for local development, So i would prefer leaving it here.
Anyhow in docs also we have mentioned it for development [here](https://github.com/commercetools/commercetools-adyen-integration/blob/master/docs/ContributionGuide.md#development).

Reply:
Makes sense. https://github.com/commercetools/commercetools-adyen-integration/pull/976/commits/bcf6dd8da410d58f83643028c9287fa79b87604d